OrbitGl/ThreadTrack.cpp

```diff
@@ -202,9 +221,16 @@ void ThreadTrack::UpdatePrimitives(uint64_t min_tick, uint64_t max_tick) {
           SetTimesliceText(timer, elapsed_us, min_x, &text_box);
         }
         batcher->AddShadedBox(pos, size, z, color, PickingID::BOX, &text_box);
+        // For boxes, we can ignore the entire timespan from start to end.
```

Comment:
Is this needed?  There is no timespan overlap on a single "depth" so in my mind this should have no effect.  Am I missing something?

Reply:
Yes, you are right. Removed.